### PR TITLE
Add more vessels from EU list

### PR DESCRIPTION
**Pull request by bitsofbits**
- Add Longliners from EU list
- Add all other vessels from EU list as "Other fishing"
- reformat using YAPF

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks good. Can you update the PR description to add that this also includes removal of all the weight-generation code (and the associated output column)?

Also, and I need to practice what I preach, we should probably be submitting something like this as three PRs for ease of review: one just running yapf (in fact you could just run it and submit directly), one removing the weight column and one adding the new vessels.

To that end, would you mind running yapf on master and just submitting, then merging the changes into this PR before submitting? It'll make the history a lot clearer.

**Comment by bitsofbits** [comment]
@seacourtaw , I broke this into two pieces, the first one just removes the column. See #5. The 
second adds the new lists.  I also ran YAPF on master first.

File: mussidae/vessel_labelling/vessel_label_mapping.py [hunk 1]
```diff
@@ -47,6 +51,8 @@ def _utf_8_encoder(unicode_csv_data):
     'Trawlers': _TRAWLER,
     'Longliners': _LONGLINER,
     'Tuna longliners': _LONGLINER,
+    # other : _OTHER_FISHING, # TODO: [bitsofbits] Find out categories an only do this if we arent'
+    #                                              sweeping up classes we are already using (pots and traps, etc)
```
Comment (seacourtaw) [inline mussidae/vessel_labelling/vessel_label_mapping.py]: Nit: Typos... :-)

File: mussidae/vessel_labelling/vessel_label_mapping.py [hunk 2]
```diff
@@ -110,44 +120,73 @@ def _hash_mmsi_to_double(mmsi, salt):
...
+            for k, v in self._mapping.items():
+                if k is other:
+                    if other_label is not None:
+                        raise ValueError("multiple other labels")
+                    other_label = v
+            self._other_label = other_label
+        return self._other_label
+
+    @property
+    def db_name(self):
```
Comment (seacourtaw) [inline mussidae/vessel_labelling/vessel_label_mapping.py]: Can you either comment this or give it a clearer name?

**Review by ghost — APPROVED** [review]
Looks good. A couple of tiny comments. Thanks!

File: mussidae/vessel_labelling/vessel_label_mapping.py (hunk 2 above)
Comment (bitsofbits) [inline mussidae/vessel_labelling/vessel_label_mapping.py]: On it!

File: mussidae/vessel_labelling/vessel_label_mapping.py (hunk 1 above)
Comment (bitsofbits) [inline mussidae/vessel_labelling/vessel_label_mapping.py]: Cleaned up.

**Pull request merged**